### ISSUE TITLE
nixops can be installed from nixpkgs again

### ISSUE THREAD

**Issue by jonasnick** [jonasnick/nix-bitcoin#3]
@nixbitcoin I don't think it's necessary anymore to install your own nixops from source. Can you check that? If so, we can remove the instructions from the tutorial.

**Comment by nixbitcoin** [comment]
Not working on my end yet. I'm getting error 
```
error: value is null while a set was expected, at /nix/store/ln25ib694xa2ydh2vxirj89pn50wbz6y-nixpkgs-19.03pre161900.61c3169a0e1/nixpkgs/pkgs/top-level/default.nix:63:5
error: evaluation of the deployment specification failed
```
Will periodically check and resubmit PR with updated README.md.

**Comment by jonasnick** [comment]
fixed by bb07fa95894866a1ffc5a753fcfdaa7fefde13b2
